jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/events/JettyAnnotatedEventDriver.java

```diff
@@ -243,6 +301,14 @@ public void onTextFrame(ByteBuffer buffer, boolean fin) throws IOException
     @Override
     public void onTextMessage(String message)
     {
+        if (LOG.isDebugEnabled())
+        {
+            if (message == null)
+                LOG.debug("onTextMessage(<null>) - events.onText={}", events.onText);
```

Comment:
It is not possible to have a null message here, it can be empty string but not null.